test/functional/test_framework/p2p.py

```diff
@@ -4,10 +4,14 @@
 # Copyright (c) 2010-2020 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
-"""Bitcoin P2P network half-a-node.
+"""Test objects for interacting with a bitcoind node over the p2p protocol.
 
-This python code was modified from ArtForz' public domain half-a-node, as
-found in the mini-node branch of http://github.com/jgarzik/pynode.
+The P2PInterface objects interact with the bitcoind nodes under test using the
```

Comment:
```suggestion
The P2PInterface classes interact with the ____ nodes under test using the
```

I think this should say "classes" instead. Also, I am not sure about `bitcoind`, since it is also possible to run the tests against the gui. Though using PACKAGE_NAME here might be a bit too abstract?